README.md

```diff
@@ -34,9 +36,41 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Quick Installation via Helm
+## Installation via Helm
+
+### Quick Installation
 
 StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then simply run the helm quick install script (or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options)
+```sh
+./scripts/quick-helm-install.sh 
+```
+The script will add the stackrox helm repository, generate an admin password, install stackrox-central-services, create an init bundle for shared secrets, and finally create one secured cluster.
+If you wish to create more secured clusters you may do so using
```

Comment:
```suggestion
To add another secured cluster, run the following:
```

This text should be placed somewhere later - the user didn't see the UI yet and we write already about another secured cluster?